modules/tinymce/src/core/main/ts/file/Conversions.ts

```diff
@@ -64,27 +65,33 @@ const parseDataUri = function (uri: string) {
   };
 };
 
-const dataUriToBlob = function (uri: string): Promise<Blob> {
-  return new Promise(function (resolve) {
-    let str, arr, i;
+const buildBlob = (type: string, data: string): Option<Blob> => {
```

Comment:
This is mostly duplicate with `imagetools` which was recombined into TinyMCE with the monorepo 🤔 
https://github.com/tinymce/tinymce/blob/develop/modules/tinymce/src/core/main/ts/file/Conversions.ts#L67

Reply:
Not sure I want to merge those in this PR but you are right there is code duplication here.